ast/cinaps/grammar.mli

```diff
@@ -1,5 +1,9 @@
-val string_of_ty : internal: bool -> Astlib.Grammar.ty -> string
+val string_of_targ : ?nodify: unit -> internal: bool -> Astlib.Grammar.targ -> string
+
+val string_of_ty : ?nodify: unit -> internal: bool -> Astlib.Grammar.ty -> string
+
 val string_of_tuple_type :
+  ?nodify: unit ->
```

Comment:
I'm not a fan of the optional `unit` argument convention. We don't "cut" the default value out of other optional types, e.g. with an optional `int` and a default of 0 we don't document it as an optional "non-zero integer". I'd prefer `?nodify:bool` and a documented default. I also think `~nodify:true` and `nodify:false` read better than `~nodify:()` and `?nodify:None`.

Reply:
Moving it back to a boolean sounds good to me!